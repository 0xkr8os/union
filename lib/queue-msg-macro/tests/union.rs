use queue_msg_macro::msg_struct;

#[msg_struct]
union Union {
    a: u32,
    b: f32,
}

fn main() {}
