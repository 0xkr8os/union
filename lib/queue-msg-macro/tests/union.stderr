error: queue-msg only supports Enum and Struct
 --> tests/union.rs:4:1
  |
4 | union Union {
  | ^^^^^
